Xamarin.Forms.Core/NativeBindingHelpers.cs

```diff
@@ -0,0 +1,145 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Reflection;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.Forms
+{
+	static class NativeBindingHelpers
+	{
+		public static void SetBinding<TNativeView>(TNativeView target, string targetProperty, BindingBase bindingBase, string updateSourceEventName) where TNativeView : class
+		{
+			if (string.IsNullOrEmpty(updateSourceEventName))
+				throw new ArgumentNullException(nameof(updateSourceEventName));
+
+			var eventWrapper = new EventWrapper(target, targetProperty, updateSourceEventName);
```

Comment:
Do we have tests for when we try to connect to a non-existing event ? Or to a non-existing property ? It should throw (a meaningful exception) in both cases.